src/accountant.rs

```diff
@@ -164,17 +177,25 @@ impl Accountant {
 
     /// Process a Transaction that has already been verified.
     pub fn process_verified_transaction(&self, tr: &Transaction) -> Result<()> {
-        self.process_verified_transaction_debits(tr)?;
-        self.process_verified_transaction_credits(tr);
-        Ok(())
+        return match self.process_verified_transaction_debits(tr) {
```

Comment:
Was this changed needed? I'm having trouble understanding what is different than the original code.